res/entities/Player.json

```diff
@@ -4,13 +4,13 @@
     {
       "componentType": "Player"
     },
-	{
+	  { 
       "componentType": "Sprite",
       "src": "entity/player_modelDefault",
-	  "origin": {
-		"x": 18,
-		"y": 48
-	  },
+	    "origin": {
+		    "x": 15,
```

Comment:
Actually, you can [do this in this very pull request](https://help.github.com/articles/committing-changes-to-a-pull-request-branch-created-from-a-fork/).